app/src/main/java/de/hhn/aib/labsw/blackmirror/dataclasses/Widget.kt

```diff
@@ -1,7 +1,25 @@
 package de.hhn.aib.labsw.blackmirror.dataclasses
 
-class Widget(name: String, x: Int, y: Int) {
+/**
+ * This class represents the widgets on the mirror.
+ *
+ * @author Selim Özdemir, Niklas Binder
+ * @version 24-05-2022
+ */
+class Widget(type: WidgetType, x: Int, y: Int) {
```

Comment:
again this should be data class (see https://kotlinlang.org/docs/data-classes.html)